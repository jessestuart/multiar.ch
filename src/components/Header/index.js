/* @flow */
import React from 'react'
import Link from 'gatsby-link'

type Props = {
  siteTitle: string,
}

const Header = ({ siteTitle }: Props) => {
  return (
    <div
      className="pv3"
      style={{
        background: '#e5f4ee',
      }}
    >
      <div
        className="flex justify-center items-center"
        style={{
          margin: '0 auto',
          maxWidth: 960,
        }}
      >
        <h1 className="ma0 mt2 f1">
          <Link
            to="/"
            className="fw3 no-underline"
            style={{
              textDecoraction: 'none',
              color: '#68465b',
              letterSpacing: '-0.15rem',
            }}
          >
            {siteTitle}
          </Link>
        </h1>
      </div>
    </div>
  )
}

export default Header
