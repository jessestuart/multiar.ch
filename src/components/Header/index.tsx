import Link from 'gatsby-link'
import React from 'react'
import { Flex, Text } from 'rebass'

const Header = ({ title }: { title: string }) => {
  return (
    <Flex bg="secondary">
      <div className="justify-center items-center center mw7">
        <h1 className="ma0 mt2 f1">
          <Link
            to="/"
            className="fw3 no-underline"
            style={{
              letterSpacing: '-0.15rem',
              textDecoration: 'none',
            }}
          >
            <Text color="primary" fontFamily="mono">
              {title}
            </Text>
          </Link>
        </h1>
      </div>
    </Flex>
  )
}

export default Header
