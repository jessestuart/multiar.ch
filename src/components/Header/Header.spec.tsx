import React from 'react'
import renderer from 'react-test-renderer'
import theme from '../../styles/Theme'
import { ThemeProvider } from 'styled-components'

import Header from '.'

test('Render Header component.', () => {
  const component = renderer.create(
    <ThemeProvider theme={theme}>
      <Header>multiar.ch</Header>
    </ThemeProvider>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
