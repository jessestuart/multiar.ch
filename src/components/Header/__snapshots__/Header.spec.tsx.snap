// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header component. 1`] = `
<div
  className="sc-bdVaJa sc-bwzfXH edpdtm"
>
  <div
    className="justify-center items-center center mw7"
  >
    <h1
      className="ma0 mt2 f1"
    >
      <a
        className="fw3 no-underline"
        href="/true"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "letterSpacing": "-0.15rem",
            "textDecoration": "none",
          }
        }
      >
        <div
          className="sc-bdVaJa sc-htpNat dtuhmt"
          color="primary"
          fontFamily="mono"
        >
          multiar.ch
        </div>
      </a>
    </h1>
  </div>
</div>
`;
